porch/func/server/server.go

```diff
@@ -48,10 +57,34 @@ func run() error {
 		return fmt.Errorf("failed to listen: %w", err)
 	}
 
-	evaluator, err := internal.NewEvaluatorWithConfig(*functions, *config)
-	if err != nil {
-		return fmt.Errorf("failed to initialize evaluator server: %w", err)
+	availableRuntimes := map[string]struct{}{
+		execRuntime: {},
+		podRuntime:  {},
+	}
+	if disableRuntimes != nil {
+		runtimesFromFlag := strings.Split(*disableRuntimes, ",")
+		for _, rt := range runtimesFromFlag {
+			delete(availableRuntimes, rt)
+		}
+	}
+	runtimes := []pb.FunctionEvaluatorServer{}
+	for rt := range availableRuntimes {
+		switch rt {
+		case execRuntime:
+			execEval, err := internal.NewExecutableEvaluator(*functions, *config)
+			if err != nil {
+				return fmt.Errorf("failed to initialize executable evaluator: %w", err)
+			}
+			runtimes = append(runtimes, execEval)
+		case podRuntime:
+			podEval, err := internal.NewPodEvaluator(*podNamespace, *wrapperServerImage)
+			if err != nil {
+				return fmt.Errorf("failed to initialize pod evaluator: %w", err)
+			}
+			runtimes = append(runtimes, podEval)
+		}
 	}
+	evaluator := internal.NewMultiEvaluator(runtimes...)
```

Comment:
We may want to at least log if no runtime is enabled...

Reply:
Added